nav2_toolkit/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>nav2_toolkit</name>
+  <version>0.1.0</version>
```

Comment:
Please match the version with the other Nav2 packages.

Also, add this package to the `navigation2/package.xml` as a dependency to link this as part of the metapackage